scripts/src/primazactl/cmd/create/namespace/common.py

```diff
@@ -72,6 +73,15 @@ def add_args_namespace(parser: argparse.ArgumentParser, type):
                 on which Primaza's Control Plane is running",
         default=None)
 
+    parser.add_argument(
+        "-j", "--primaza-identity-namespace",
+        dest="primaza_identity_namespace",
+        required=False,
+        help=f"name to be used for the WorkerNamespace that will be created,\
+             default: {WORKER_NAMESPACE}.",
```

Comment:
```suggestion
        help=f"namespace used for hosting the service account shared with\
             Primaza's Control Plane. Default: {WORKER_NAMESPACE}.",
```

Reply:
The _ is there is all the other options so thought to keep the same way